react/components/UI/Text/index.js

```diff
@@ -0,0 +1,21 @@
+import styled, { css } from 'styled-components';
+import { display, space, textColor, fontFamily, fontSize, lineHeight, textAlign } from 'styled-system';
+
+import { preset } from 'react/styles/functions';
+import { antialiased, neutralMarginsY } from 'react/styles/mixins';
+
+export const mixin = css`
+  ${display}
+  ${space}
+  ${preset(fontFamily, { font: 'sans' })}
+  ${preset(fontSize, { f: 5 })}
+  ${preset(lineHeight, { lineHeight: 1 })}
+  ${preset(textColor, { color: 'gray.base' })}
```

Comment:
This is the only real gotcha I've run into. When you specify preset values; you'll want to use the shorthand aliases.

I explain here: https://github.com/jxnblk/styled-system/pull/37#issuecomment-400699999 —there might be a way around this by inspecting the function propTypes inside of the preset function I wrote. Or there might be some other obvious way to deal with this issue. But for now it's a minor inconvenience.

https://github.com/jxnblk/styled-system/blob/master/src/styles.js serves as a reference for aliases.

Reply:
Another option would be to expose our own wrapped functions that accept presets.